app/presenters/publishing_api/editionable_worldwide_organisation_presenter.rb

```diff
@@ -90,6 +90,8 @@ def home_page_offices
     end
 
     def contacts
+      return [] unless item.main_office&.contact || item.home_page_offices&.map(&:contact)&.any?
```

Comment:
would it be good to pull `item.main_office&.contact` and `item.home_page_offices&.map(&:contact)` into variables? As is they're used in a few places, not just in this file and am thinking it might be a bit easier to read

Reply:
Done 👍